src/Chart.js

```diff
@@ -0,0 +1,105 @@
+import React, {useState, useEffect} from 'react';
+import { LineChart, Line, CartesianGrid, XAxis, YAxis, Tooltip } from 'recharts';
+import { retrieveLogDatabase } from './DatabaseFunctions';
+import { Dropdown, Button } from 'semantic-ui-react';
+
+function kemmlerEquation(weight, reps) {
+  return weight * (0.988 + 0.0104*reps + 0.0019*reps*reps - 0.0000584*reps*reps*reps);
+}
+
+function createCharts(exerciseName) {
+  const LogDatabase = Object.entries(retrieveLogDatabase());
+  let dailyPRs = [];
+
+  for (let i = 0; i < LogDatabase.length; i++) {
+    let Log = LogDatabase[i][1];
+    let currentPRs = {
+      estimated: 0,
+      tested: 0
+    }
+
+    for (let j = 0; j < Log.length; j++) {
+      let exercise = Log[j];
+      let estimated1RM = Math.floor(kemmlerEquation(exercise.Weight, exercise.Reps));
+
+      if (exerciseName !== exercise.exerciseName) continue;
```

Comment:
Is there anything preventing you from adding the same exercise twice to the log?